clientv3/example_watch_test.go

```diff
@@ -70,7 +70,7 @@ func ExampleWatcher_watchWithRange() {
 	}
 	defer cli.Close()
 
-	// watches within ['foo1', 'foo4'), in lexicographical order
```

Comment:
I believe this is correct. https://github.com/coreos/etcd/blob/master/clientv3/op.go#L195

/cc @heyitsanthony